advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+
+![Target Instance user and password](Images/targetuserpassword.png)
+
+12. Click Ok on the Prepare User Database screen.
+
+![Prepare User Database](Images/prepareuserdatabase2.png)
+
+13. A success message will be displayed once the  Prepare User Database is successful.
+
+![Prepare User Database Completed successfully](Images/doneprepareuserdatabase.png)
+
+## Create Distribution on Connect CDC Director
+
+The following steps demonstrate the creation of Distribution which will define how data will be replicated from source to target. In our case from EDB Postgres Advanced Server source to EDB Postgres Advanced Server target.
+
+1. Open Connect CDC Director.
+
+2. Right click on Distributions and select New Distribution.
+
+3. Enter Distribution information e.g distribution name, description etc and click ok.
+
+![New Distribution](Images/newdistribution.png)
+
+4. Go to Source DBMS Server —> Tables —> Available Tables —> Public (Schema name).
+
+5. Select the table(s) to replicate using the checking box in front of them and right click and select Select for Distribution —> All Checked Tables.
+
+![Select Tables for Replication](Images/selecttableforreplication.png)
+
+6. On the Distributed Tables Screen, select the Target Server under “Select one or more target servers” and select “Create tables on target” option and select the distribution under “Select a Distribution” option and click ok.
+
+![Distributed Tables](Images/distributetables.png)
+
+7. On Connect CDC Director Screen, select Yes.
+
+![CDC Director Screen](Images/connectcdc.png)
+
+8. On the Target Tables Screen, enter the Target Schema against each table and Click Next.
+
+![Target Tables Screen](Images/targettablescreen.png)
+
+9. On Target Server Login Details Screen, enter EDB Postgres Advanced Server Target Instance user and password and click test connect. Once successful, click Add and then click Create.
+
+![Target Server Login Details Screen](Images/targetserverlogin.png)
+
+10. Once done, click Finish.
+
+11. To confirm the tables are created on Target Server, click the Target DBMS Server —> Tables —> Receiving Tables, the list of the tables will be displayed which are available on Source DBMS Server.
+
+![Confirm Tables Are Created on Target Server](Images/confirmtablesontarget.png)
+
+12. Open pgAdmin and connect to EDB Postgres Advanced Server Target Instance, and under Schema public, the tables will be created.
+
+![pgAdmin](Images/pgAdmin.png)
+
+13. Mappings of tables setup for replication from EDB Postgres Advanced Server Source Instance to  EDB Postgres Advanced Server Target Instance will be created under Distributions —> Mappings.
+
+![Mappings of Tables](Images/tablemappings.png)
+
+14. Open Distributions —> Newly Created Distribution —> Requests —> New Request —> Copy. Copy will copy/replicate all the data in tables from EDB Postgres Advanced Server Source Instance to  EDB Postgres Advanced Server Target Instance.
```

Comment:
Open `Distributions` —> `Newly Created Distribution` —> `Requests` —> `New Request` —> `Copy`. `Copy` will copy/replicate all the data in the tables from the EDB Postgres Advanced Server Source Instance to the EDB Postgres Advanced Server Target Instance.

Reply:
Done, guide updated.